t/op/ver.t

```diff
@@ -267,5 +267,27 @@ sub { $_[0] = v3;
 *{"\3"} = *DATA;
 is( (readline v3), "This is what we expect to see!\n", "v-strings even work in Mordor" );
 
+$DOWARN = 0; # disable warnings just for the following test
```

Comment:
Since `$DOWARN` appears to be a package variable it would be better to surround the entire test sequence in a block, and put

```perl
local $DOWARN = 0;
```
at the top, as this will avoid the need to remember to reset it to 1 again afterwards.

Reply:
I believe I've resolved these in `ac4a599`